ddl/ddl.go

```diff
@@ -709,6 +709,12 @@ func setDDLJobQuery(ctx sessionctx.Context, job *model.Job) {
 // - context.Cancel: job has been sent to worker, but not found in history DDL job before cancel
 // - other: found in history DDL job and return that job error
 func (d *ddl) DoDDLJob(ctx sessionctx.Context, job *model.Job) error {
+	if mci := ctx.GetSessionVars().StmtCtx.MultiSchemaInfo; mci != nil {
```

Comment:
At the ddl_api level, please add a parallel processing test:
when processing add columns (Multi-Schema change) operation, other DDL requests came in

Reply:
Do you mean the test like [this](https://github.com/pingcap/tidb/pull/32899/files#diff-4c5da8cca412bedfcaaa5ebad4080789024a8bd7a37d6033387d40645fc2db45R268)? This test puts 2 multi-schema change DDLs to the job queue at the same time.

Let me add it in the next PR.

Reply:
I'm not sure what "putTheSameDDLJobTwice" does, but it's a similar test